.gitignore

```diff
@@ -69,46 +87,40 @@ PCbuild/arm64/
 PCbuild/obj/
 PCbuild/win32/
 Tools/unicode/data/
-.purify
-__pycache__
-autom4te.cache
-build/
-buildno
-config.cache
-config.log
-config.status
-config.status.lineno
-core
-!Tools/msi/core/
-db_home
-.hg/
-.idea/
-ipch/
-libpython*.a
-libpython*.so*
-libpython*.dylib
-libpython*.dll
-platform
-pybuilddir.txt
+/autom4te.cache
+/build/
+/buildno
+/config.cache
+/config.log
+/config.status
+/config.status.lineno
+/db_home
```

Comment:
This is a holdover from the `bsddb` package, and can go away (so can the explicit callout of `db_home` in Lib/test/libregrtest/runtest.py).